lib/rubocop/cop/sorbet/sigils/has_sigil.rb

```diff
@@ -15,7 +15,7 @@ module Sorbet
       #
       # If a `MinimumStrictness` level is specified, it will be used in offense messages and autocorrect.
       class HasSigil < ValidSigil
-        @registry = Cop.registry # So we can properly subclass this cop
+        @registry = Registry.global # So we can properly subclass this cop
```

Comment:
Is this even meaningful anymore? The comment seems to indicate that subclasses would not inherit the `@registry` instance variable and would thus break `Cop.registry` for subclass. Since we are moving to `Registry.global`, it feels like this should no longer be necessary.